Assets/_BForBoss/Scripts/UserInterface/SettingsViewBehaviour.cs

```diff
@@ -7,30 +7,49 @@ namespace BForBoss
     public class SettingsViewBehaviour : MonoBehaviour
     {
         [SerializeField] private Button _backButton = null;
+
+        private InputSettingsViewBehaviour _inputSettingsView = null;
+        private SetUsernameViewBehaviour _setUsernameView = null;
+        private LeaderboardPanelBehaviour _leaderboardView = null;
+        private TabbedPanelViewBehaviour _tabbedPanelViews = null;
         
-        private InputSettingsViewBehaviour InputSettingsView => GetComponentInChildren<InputSettingsViewBehaviour>();
-        private SetUsernameViewBehaviour SetUsernameView => GetComponentInChildren<SetUsernameViewBehaviour>();
-        private TabbedPanelViewBehaviour TabbedPanelViews => GetComponentInChildren<TabbedPanelViewBehaviour>();
```

Comment:
Certain Views can't be found at times, since the gameobject is not active. So im just setting them on awake